chevah_build

```diff
@@ -159,6 +159,13 @@ case $OS in
         export CC="cc"
         export CXX="CC"
         export MAKE="gmake"
+
+        # The package on Solaris 10 does not compile with CC, we use GCC.
```

Comment:
It failed to me like this at some points during my tests, as if trying to compile a 64bit package on a 32bit environment (note this is testing master branch on the Solaris 10 u11 x64 just now):
https://chevah.com/buildbot/builders/python-package-solaris-10/builds/94/steps/build/logs/stdio

At this moment If you test this branch with cc you would get this:
https://chevah.com/buildbot/builders/python-package-solaris-10u3-x86/builds/45/steps/build/logs/stdio

So it is only working with gcc from what I could test.